### DISCUSSION TITLE
Powerpoint Export - Missing Slide Title

### DISCUSSION THREAD

**Discussion by hasantahir** [marp-team/marp#380] [Q&A] [answered]
First, thank you for making Marp available to us. On the accessibility front, Marp creates fully accessible HTML pages. 

Some students prefer PowerPoint slides, perhaps due to the ease of making notes. Currently, Accessibility Inspector in Powerpoint throws some errors that the generated slides have missing titles. Would it be possible to map the page heading to a slide title? I use the following command to generate a PPTX:

`marp  --html --pptx  --bespoke.transition lecture\ 6.md -o lecture6.pptx`

Thanks.

**Comment by yhatt** [comment, marked as answer]
Marp's PPTX export is made for doing presentation with the familiar tool, and the PowerPoint document has consisted with pre-rendered slides as background images. There are no any other placeholder elements, such as the text of slide title.

For getting accesible PPTX document, export Marp Markdown to PDF once, and convert it to PPTX by using [Adobe's online tool](https://www.adobe.com/acrobat/online/pdf-to-ppt.html). In each slide pages, `# Heading` (`<h1>` in HTML) would be recognized as the slide title.

### Related

- #82 
- marp-team/marp-vscode#139

**Reply by hasantahir** [reply]
Thanks for replying.
